src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -146,4 +158,32 @@ public void delete(@PathVariable Long id) {
         }
     }
 
+    /**
+     *Method to search a book by the following variables
```

Comment:
Can you add a space between * and the comment please?

Reply:
corrected